web-app/js/portal/data/DataCollection.js

```diff
@@ -67,6 +67,16 @@ Portal.data.DataCollection = function() {
         return applicableDownloadOptions;
     };
 
+    constructor.prototype.getFilterParams = function() {
+        var layer = this.getSelectedLayer();
```

Comment:
I'm surprised this works? `getSelectedLayer()` shouldn't be in `DataCollection` any more...